python/ufoff.py

```diff
@@ -0,0 +1,114 @@
+
+from ufoLib2 import Font
+import flatbuffers
+from FlatFont.Ufo import FontInfo
+from enum import IntEnum
+
+SCALARS = [int, float, bool] # Other scalars needed # Enum needed
+NON_SCALARS = [str, list] # Other none-scalars needed
+
+def Capital(string: str): # seek equivalent buit-in function
+    '''Capitalizes the string without affecting other characters'''
+    return string[0].upper() + string[1:]
+
+def StartAttrVector(attr: str):
```

Comment:
you're annotating the parameters but not the return value, why not both? in this case it would be `StartAttrVector(attr: str) -> str:`